src/modules/Builder/components/ToggleItemContainer/ToggleItemContainer.styles.ts

```diff
@@ -15,9 +15,8 @@ export const StyledItemOptionContainer = styled(StyledFlexColumn, shouldForwardP
   margin-bottom: ${theme.spacing(2.4)};
 `;
 
-export const StyledItemOption = styled(StyledItemOptionContainer, shouldForwardProp)`
-  padding: ${({ uiType }: { uiType: ToggleContainerUiType }) =>
-    theme.spacing(1.6, 2.4, 1.6, uiType === ToggleContainerUiType.Item ? 3.4 : 2.4)};
+export const StyledItemOption = styled(StyledItemOptionContainer)`
+  padding: ${theme.spacing(1.6, 2.4, 1.6, 2.4)};
```

Comment:
are you sure this doesn't change right styles for flanker?
(this changes were made for flanker)